flake.nix

```diff
@@ -28,6 +28,26 @@
             overlays = [
               (final: prev: {
 
+                wasm-bindgen-cli = final.rustPlatform.buildRustPackage rec {
+                  pname = "wasm-bindgen-cli";
+                  version = "0.2.89";
+                  hash = "sha256-IPxP68xtNSpwJjV2yNMeepAS0anzGl02hYlSTvPocz8=";
+                  cargoHash = "sha256-pBeQaG6i65uJrJptZQLuIaCb/WCQMhba1Z1OhYqA8Zc=";
+
+                  src = final.fetchCrate {
+                    inherit pname version hash;
+                  };
+
+                  nativeBuildInputs = [ final.pkg-config ];
+
+                  buildInputs = [ final.openssl ] ++ lib.optionals stdenv.isDarwin [ final.curl final.Security ];
+
+                  nativeCheckInputs = [ final.nodejs ];
+
+                  # tests require it to be ran in the wasm-bindgen monorepo
+                  doCheck = false;
```

Comment:
`nativeCheckInputs` feels redundant, why isn't it? (just trying to improve my Nix understanding)

Reply:
Huh. I guess it makes sense to have inputs that are only used in the check phase (which here is disabled anyway, but I guess it was added for completeness?). I copied the whole thing from the nixpkgs.